packages/ember-htmlbars/lib/keywords/component.js

```diff
@@ -22,6 +22,8 @@ export default {
       return;
     }
 
+    hash.isComponentHelper = true;
```

Comment:
I doubt it very much that this is the correct way to inform the `ComponentNodeManager` that the component which shall be rendered is declared using the `{{component}}` helper. What would be the correct way?